c7n/schema.py

```diff
@@ -190,6 +190,7 @@ def generate(resource_types=()):
                 'end': {'format': 'date-time'},
                 'resource': {'type': 'string'},
                 'max-resources': {'type': 'integer'},
+                'max-resources-percent': {'type': 'number'},
```

Comment:
should max-resources and max-resources-percent be 'OneOf'? I can't think of a valid reason to include both in a policy

Reply:
also, consider adding max/min to the schema:

```python
   'max-resources-percent': {'maximum': 100, 'minimum': 0, 'type': 'number'}
```